actionview/Rakefile

```diff
@@ -8,7 +8,7 @@ task :package
 # Run the unit tests
 
 desc "Run all unit tests"
-task test: ["test:template", "test:integration:action_pack", "test:integration:active_record"]
+task test: ["test:template", "test:integration:action_pack", "test:integration:active_record", "test:integration:ujs"]
```

Comment:
I think it is worth to run the integration tests in a different job as action view. I think they will be flaky so I don't want to make all the ruby tests to fail because the integration tests failed.